pkg/tests/ossm/route_prevent_additional_ingress_test.go

```diff
@@ -0,0 +1,73 @@
+package ossm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	. "github.com/maistra/maistra-test-tool/pkg/util/test"
+	"github.com/maistra/maistra-test-tool/pkg/util/version"
+)
+
+func TestRoutePreventAdditionalIngress(t *testing.T) {
+	NewTest(t).Id("T48").Groups(Full).MaxVersion(version.SMCP_2_5).Run(func(t TestHelper) {
+
+		ER := "extra-routes"
+
+		t.Cleanup(func() {
+			oc.DeleteFromString(t, ER, smcp_additionalIngress)
+			oc.DeleteNamespace(t, "extra-routes")
```

Comment:
Here, better to use the same variable defined above:
`oc.DeleteNamespace(t, extraRoutesNS)`

Reply:
@fjglira : Thanks for checking. Will update it.